cli/user/src/main/java/io/pravega/cli/user/kvs/KeyValueTableCommand.java

```diff
@@ -35,12 +44,8 @@
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
-import lombok.Cleanup;
-import lombok.Data;
-import lombok.NonNull;
-import lombok.val;
 
-abstract class KeyValueTableCommand extends Command {
+public abstract class KeyValueTableCommand extends Command {
```

Comment:
Why did you make all of these public? If just for testing, shouldn't the tests be in the same package?

Reply:
I have refactored the code by placing each set of commands into a separate package. As the Runner uses all the commands, and it is located in the parent package, I had to change the visibility of this and other classes. Before, everything was on the same package, so that change was not necessary.